modules/osdk-upgrading-v180-to-v1100.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-upgrading-projects.adoc
+
+:osdk_ver: v1.10.1
+:osdk_ver_n1: v1.8.0
+
+[id="osdk-upgrading-v180-to-v1100_{context}"]
+= Upgrading projects for Operator SDK {osdk_ver}
+
+The following upgrade steps must be performed to upgrade an existing Operator project for compatibility with {osdk_ver}.
+
+.Prerequisites
+
+- Operator SDK {osdk_ver} installed
+- Operator project that was previously created or maintained with Operator SDK {osdk_ver_n1}
+- If the project was created with a version prior to {osdk_ver_n1}, completion of the link:https://docs.openshift.com/container-platform/4.8/operators/operator_sdk/osdk-upgrading-projects.html#osdk-upgrading-v130-to-v180_osdk-upgrading-projects[Upgrading projects for Operator SDK v1.8.0] procedure 
```

Comment:
Do you expect most of your users to have already performed this upgrade step? I might keep the old instructions for a few more versions and be clearer about which versions they apply to in the titles. While this link makes sense, I'd still avoid it. It'll still be an unoptimized experience for people reviewing this topic on the portal.

Reply:
Expect users to have already performed this step, so it was just a little bonus here. Just gonna remove the bullet here tho, but leave the "Additional resources" link down below.